pandas/tseries/frequencies.py

```diff
@@ -19,30 +19,17 @@
 from pandas.util._decorators import cache_readonly, deprecate_kwarg
 import pandas.tseries.offsets as offsets
 
-from pandas._libs import lib, tslib
+from pandas._libs import tslib
 from pandas._libs.tslib import Timedelta
-from pandas._libs.tslibs.frequencies import (  # noqa
-    get_freq_code, _base_and_stride, _period_str_to_code,
-    _INVALID_FREQ_ERROR, opattern, _lite_rule_alias, _dont_uppercase,
-    _period_code_map, _reverse_period_code_map)
+import pandas._libs.tslibs.frequencies as libfreqs
+from pandas._libs.tslibs.frequencies import get_freq_code
+from pandas._libs.tslibs.frequencies import (  # noqa, semi-public API
+    get_freq, get_base_alias, get_to_timestamp_base,
+    FreqGroup,
```

Comment:
add to the list to remove these from the 'public' api (which basically means removing and adding to the whatsnew note) (maybe can leave is_subperiod/is_superperiod I guess)

Reply:
the rest are impl details.